virtme/guest/virtme-sshd-script

```diff
@@ -70,6 +70,7 @@ AuthorizedKeysFile ${SSH_AUTH_KEYS}
 PubkeyAuthentication yes
 UsePAM yes
 PrintMotd no
+StrictModes no
```

Comment:
because virtme is used for the development, and should be only available locally, I guess that's fine